### PR TITLE
Add manifest for discord-delete

### PR DESCRIPTION
**Pull request by cedws**

### CODE REVIEW

File: bucket/discord-delete.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+    "version": "1.3.1",
+    "description": "Tool to delete Discord message history",
+    "homepage": "https://github.com/adversarialtools/discord-delete",
+    "license": "GPL-3.0-or-later",
```
Comment (Ash258) [inline bucket/discord-delete.json:5]: Github licenses are not explicitly `-or-later`. Use `-only`

File: bucket/discord-delete.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
+{
+    "version": "1.3.1",
+    "description": "Tool to delete Discord message history",
```
Comment (Ash258) [inline bucket/discord-delete.json:3]: > Be warned that using this tool could result in the termination of your account,

I do not like expose this tool to scoop users.

**Review by Ash258 — CHANGES_REQUESTED** [review]

File: bucket/discord-delete.json (hunk 2 above)
Comment (cedws) [reply bucket/discord-delete.json:3]: No comment. It's just how the tool works. I see no reason why this should be excluded from the bucket.

Comment (Ash258) [reply bucket/discord-delete.json:3]: /unsubscribe

Allowing users to install the tool, which will terminate their accounts without prior notice is dangerous and does not make sense.

I will not be the one who will merge this and be responsible for this.

Comment (cedws) [reply bucket/discord-delete.json:3]: That's not what it says. It says that it *could* result in termination, and to date, this has never happened to a user of the tool.
It's a disclaimer to warn that it is not risk-free.

**Comment by cedws** [comment]
@Ash258 Could you kindly label this for review from another maintainer? If not, this should be closed.

**Pull request closed without merging**